xds/internal/balancer/clusterresolver/configbuilder_test.go

```diff
@@ -72,7 +73,10 @@ var (
 	}
 
 	noopODCfg = outlierdetection.LBConfig{
```

Comment:
Can't do anything about it. The gRFC specifically states "cds generates with all fields unset" for the no-op. Thus, the way our system is set up now, it is literally impossible to uphold the balancer API while keeping all fields unset (will always call ParseConfig() and get defaults). The important part is sre and fpe == nil, which keeps the picker from doing unnecessary counting on the critical rpc path. I talked to Michael about this and he says it's fine. I would like the fields to stay unset, but unfortunately can't uphold API and keep fields unset. Also can't set to zero values because that is different that the language of the gRFC, which states "all fields unset".

Reply:
Wait how is this config used in these tests, then?  It looked like they were inputs, which means they could be completely empty, right?  Or are you trying to make the input test data match real-world input data?

Reply:
The latter.